libs/custom-paper-handler/src/bits.ts

```diff
@@ -348,6 +353,12 @@ export function Uint8ToBinaryArray(value: Uint8): BinaryArray {
   return bitArray;
 }
 
+export function arrayBufferToHexString(buffer: ArrayBuffer): string {
```

Comment:
Probably useful for debugging, but not used anywhere in this PR. Consider removing.

Reply:
https://github.com/votingworks/vxsuite/pull/3756/commits/eb2ca2aa84cb657a35727fcbf2498525601d48fc